config/locales/es.yml

```diff
@@ -195,6 +195,22 @@ es:
       add_site: Agregar sitio
       index:
         title: Socios de VITA
+    organizations:
```

Comment:
Nitpicky nitpick, but I think most of these don't need quote marks around it.

Reply:
:) True - I think RubyMine added them for me, and I didn't stop it.

It might be a good time to run i18n-tasks normalize anyway immediately after this gets merged. I've validated locally (in the past 30 seconds) that it will remove the unnecessary quotation marks.